roles/webview/templates/etc/nginx/sites-available/webview

```diff
@@ -7,76 +11,16 @@ server {
     try_files       $uri $uri/ /index.html;
     expires         -1;
 
+    if ($rex_uri != "no-match") {
+        return 302 https://{{ rex_domain }}$rex_uri;
+    }
+
     location /ping {
         add_header Content-Length 0;
         add_header Content-Type text/plain;
         return 200;
     }
 
-{% if groups.authoring|length > 0 %}
```

Comment:
It isn't clear to me why this and all lines below this are getting deleted in this PR...

Reply:
It's cleanup. cnx-deploy is a "rebase and merge" repo. So the PRs here are a little looser since it's really the commits that are being merged rather than the whole PR.

Reply:
resolved in #1103